### PR TITLE
Kepori Polishing pt.1

### PR DESCRIPTION
**Pull request by PiperDoots**
## About The Pull Request

Adds tail feathers, a customisation option missing from the current implementation.  
Tail feathers currently use the same color as body feathers.
Fixed some of the plumage sprites clashing if you made it a different color from the base.
Fixes some spelling mistakes found in the UI and shuffles around some of the options.
Made the Kepori lore blurb no longer clash with current lore.

Most of the original plumage sprites were made by Honk#2574

## Why It's Good For The Game

The current Kepori implementation is a bit rough around the edges, this fixes some of those issues, such as UI asking if you want to change your face.
Also more customisation is never a bad thing.  

jary
UI

## Changelog
:cl: 
add: Added Kepori tail feathers
imageadd: Kepori plumage now works when it's a different color from the base
imageadd: Added Kepori tail feathers
spellcheck: Fixed the text of various UI elements
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
bir ds

**Review by triplezeta — APPROVED** [review]
i love birds so much

**Comment by PiperDoots** [comment]
Same,,,

**Comment by Latentish** [comment]
Bird up

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -2587,9 +2590,27 @@
 	name = "Wings"
 	icon_state = "wings"
 
-/datum/sprite_accessory/kepori_body_feathers/none
+/datum/sprite_accessory/kepori_tail_feathers/none
```
Comment (Fikou) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: These need to be moved under their parent type

**Review by MarkSuckerberg — APPROVED** [review]

**Pull request merged**